openid-data-classes/src/commonMain/kotlin/at/asitplus/openid/RelyingPartyMetadata.kt

```diff
@@ -112,8 +112,16 @@ data class RelyingPartyMetadata(
      * and Verifiers all understand the new format.
      */
     @SerialName("vp_formats")
+    @Deprecated("OID4VP Draft 27: Renamed to vp_formats_supported", ReplaceWith("vpFormatsSupported"))
     val vpFormats: FormatHolder? = null,
 
+    /**
+     * OID4VP 1.0: REQUIRED when not available to the Wallet via another mechanism. As defined in Section 11.1.
+     * An object containing a list of name/value pairs, where the name is a Credential Format Identifier and the value defines format-specific parameters that a Verifier supports.
+     */
+    @SerialName("vp_formats_supported")
+    val vpFormatsSupported: FormatHolder? = null,
```

Comment:
Use this new property in addition to `vpFormats` in `PresentationFaictory.createPresentation`